phc/services/genomics.py

```diff
@@ -259,6 +259,26 @@ def list_tests(self, project_id: str, patient_id: str) -> ApiResponse:
             http_verb="GET",
         )
 
+    def get_test(self, project_id: str, test_id: str) -> ApiResponse:
+        """Get test by project and test id
+
+        Parameters
+        ----------
+        project_id : str
+            The project ID
+        test_id : str
+            The Test ID
+
+        Returns
+        -------
+        ApiResponse
+            The list tests response
```

Comment:
I'm assuming this is a copy typo.
```suggestion
            The get test response
```